LiteNetLib/Utils/NetDataReader.cs

```diff
@@ -685,7 +685,7 @@ public bool TryGetBytesWithLength(out byte[] result)
             if (AvailableBytes >= 4)
             {
                 var length = PeekInt();
-                if (AvailableBytes >= length + 4)
+                if (length > 0 && AvailableBytes >= length + 4)
```

Comment:
Zero length buffer is acceptable thing in many places - so it maybe there should be `length >= 0`

Reply:
well... it depends on the approach and overall architecture i'm not really familiar with... i'm ok with either =)